.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,33 @@
-For plugin updates, please indicate which repos this should be built into:
+If your package needs to be released within one or more release streams, and/or distributions, please open PRs to each of those branches respectively. The easiest way to do this is to make the initial commit for the mainline branch (e.g. rpm/develop or deb/develop) and then cherry pick the commit hash onto each subsequent branch.
```

Comment:
```suggestion
If your package needs to be released within one or more release streams, and/or distributions, please open PRs to each of those branches respectively. The easiest way to do this is to make the initial commit for the mainline branch (e.g. rpm/develop or deb/develop) and then cherry pick the commit onto each subsequent branch.
```

"cherry-pick the commit hash" sounds weird to me, as you technically picking the commit, not its hash :)